get.php

```diff
@@ -36,7 +36,7 @@
 	"devtoolkit" => "https://github.com/ajborka/nvda_developer_toolkit/releases/download/20.04/developerToolkit-20.04.nvda-addon",
 	"devtoolkit-dev" => "https://github.com/ajborka/nvda_developer_toolkit/releases/download/20.04/developerToolkit-20.04.nvda-addon",
 	"dw" => "https://github.com/abdel792/dayOfTheWeek/releases/download/v19.12/dayOfTheWeek-19.12.nvda-addon",
-	"dw-dev" => "http://cyber25.free.fr/nvda-addons/dayOfTheWeek-19.12-dev.nvda-addon",
+	"dw-dev" => "https://github.com/abdel792/dayOfTheWeek/releases/download/v19.12/dayOfTheWeek-19.12.nvda-addon",
```

Comment:
It looks like this has the same issue. Perhaps this is fine and @abdel792 is ok with both stable and dev versions of this addon being the same?